content/sensu-go/5.6/api/overview.md

```diff
@@ -120,6 +121,79 @@ Content-Type: application/json
+
+The Sensu API supports filtering for all GET endpoints that return an array. You can filter resources based on their labels with a label selector using the `labelSelector` query parameter and on certain pre-determined fields with a field selector using the `fieldSelector` query parameter.
+
+For example, the following request filters the response to only include resources that have a label entry `region` with the value `us-west-1`. We will use the flag `--data-urlencode` in curl so it encodes the query parameter for us, in conjunction with the `-G` flag to it appends the data to the URL.
+
+{{< highlight shell >}}
+curl -H "Authorization: Bearer $TOKEN" http://127.0.0.1:8080/api/core/v2/checks -G \
+--data-urlencode 'labelSelector=region == "us-west-1"'
+{{< /highlight >}}
+
+#### Label selector
+
+A label selector can use any label attributes to group a set of resources. All resources support labels within the metadata object. For example, see [entities metadata attributes][8].
+
+#### Field selector
+
+A field selector can use certain fields of resources to organize and select subsets of resources. Here's the list of available fields.
+
+| Resource      | Fields      |
+| ----------- | ----------- | 
+| Asset | `asset.name` `asset.namespace` `asset.filters` |
+| Check | `check.name` `check.namespace` `check.handlers` `check.publish` `check.round_robin` `check.runtime_assets` `check.subscriptions`|
+| ClusterRole | `clusterrole.name` |
+| ClusterRoleBinding | `clusterrolebinding.name` `clusterrolebinding.role_ref.name` `clusterrolebinding.role_ref.type`|
+| Entity | `entity.name` `entity.namespace` `entity.deregister` `entity.entity_class` `entity.subscriptions` |
+| Event | `event.name` `event.namespace` `event.check.handlers` `event.check.publish` `event.check.round_robin` `event.check.runtime_assets` `event.check.status` `event.check.subscriptions` `event.entity.deregister` `event.entity.entity_class` `event.entity.subscriptions` |
+| Extension | `extension.name` `extension.namespace` |
+| Filter | `filter.name` `filter.namespace` `filter.action` `filter.runtime_assets` |
+| Handler | `handler.name` `handler.namespace` `handler.filters` `handler.handlers` `handler.mutator` `handler.type`| 
+| Hook | `hook.name` `hook.namespace` |
+| Mutator | `mutator.name` `mutator.namespace` `mutator.runtime_assets` |
+| Namespace | `namespace.name` |
+| Role | `role.name` `role.namespace` |
+| RoleBinding | `rolebinding.name` `rolebinding.namespace` `rolebinding.role_ref.name` `rolebinding.role_ref.type`|
+| Silenced | `silenced.name` `silenced.namespace` `silenced.check` `silenced.creator` `silenced.expire_on_resolve` `silenced.subscription` |
+| User | `user.username` `user.disabled` `user.groups` |
+
+#### Supported operators
+
+There's two _equality-based_ operators supported, `==` (equality) and `!=` (inequality). For example, the following requirements are possible:
```

Comment:
I think the term "requirements" is a bit confusing. Is there another term we could use? Maybe "statements"?